app/models/prosecution_case.rb

```diff
@@ -36,35 +37,13 @@ def to_builder
       prosecution_case.statementOfFactsWelsh statementOfFactsWelsh
       prosecution_case.breachProceedingsPending breachProceedingsPending
       prosecution_case.appealProceedingsPending appealProceedingsPending
-      prosecution_case.defendants Jbuilder.new.array! defendants_builder
-      prosecution_case.caseMarkers Jbuilder.new.array! markers_builder if markers.present?
+      prosecution_case.defendants array_builder(defendants)
+      prosecution_case.caseMarkers array_builder(markers)
       if split_prosecutor_case_references.present?
-        prosecution_case.splitProsecutorCaseReferences Jbuilder.new.array! split_prosecutor_case_references_builder
+        prosecution_case.splitProsecutorCaseReferences split_prosecutor_case_references.map(&:split)
       end
       prosecution_case.mergedProsecutionCase merged_prosecution_case.to_builder if merged_prosecution_case.present?
-      prosecution_case.linkedProsecutionCases Jbuilder.new.array! linked_prosecution_cases_builder if linked_prosecution_cases.present?
+      prosecution_case.linkedProsecutionCases linked_prosecution_cases.ids if linked_prosecution_cases.present?
```

Comment:
Same with this bit of logic? Arguably not but would like to know what you think.